docs/views/frontend_goblitz.md

```diff
@@ -0,0 +1,163 @@
+# FrontEnd development via GoBlitz
+
+## Assets
+
+Assets (images, gifs, css, js files etc) are served from public folder. The relative path is `/assets`.
+
+You can look an example of importing assets in the [Welcome Page Code](../../views/mainPage/welcome.html).
```

Comment:
And here if you plan to change folder name by my comment, this file should be checked and maybe other docs as here the folder names are the same with camel case.

Reply:
Yhea, fixed them in [7b6d62c](https://github.com/KostLinux/GoBlitz/pull/10/commits/7b6d62cbeeca37deb2e44f942598fe2b9ca6b21a)